components/AboutPagesCard/AboutPagesCard.module.css

```diff
@@ -0,0 +1,17 @@
+.header {
+  background-color: var(--bs-blue);
+  color: white;
+  width: 13rem;
+  height: 3rem;
+  transform: translate(-15%, -40%);
+  /*I'm not sure how to get the border-radius to override the styles
+    coming from bootstrap.
+    Using an ID would work but given this element will be repeated multiple times,
+    I'm not sure how to implement it other than the temporary !important fix.*/
+  border-radius: 0 5rem 5rem 0 !important;
```

Comment:
It seems to work moving this into its own selector, following the selector used by bootstrap:

```css
.header:first-child {
  border-radius: 0 5rem 5rem 0;
}
```

Reply:
Oh ya I see!  Yes that works.  I had tried some other solutions that hadn't.